internal/common/host_utils.go

```diff
@@ -41,3 +49,18 @@ func GetEventSeverityFromHostStatus(status string) string {
 		return models.EventSeverityInfo
 	}
 }
+
+func ValidateHostname(hostname string) error {
```

Comment:
maybe better to have it in validator? sounds like better place for me

Reply:
Validators usually run on values that are already in the database.  Here I think we want to prevent adding invalid hostname before it is added to the database.

Reply:
i just think better to have all validating functions in one place, so if one day we will want to validate hostname on running host it will be easier to find :)